api/routes.go

```diff
@@ -2,13 +2,15 @@ package api
 
 import (
 	"github.com/go-macaron/binding"
+	"github.com/go-macaron/gzip"
 	"github.com/raintank/metrictank/api/middleware"
 	"github.com/raintank/metrictank/api/models"
 	"gopkg.in/macaron.v1"
 )
 
 func (s *Server) RegisterRoutes() {
 	r := s.Macaron
+	r.Use(gzip.Gziper())
```

Comment:
all of our response data are json or msgp, both of which contain lots of text, so they compress really well. json is all text, and msgp uses text for all struct field names.